### PR TITLE
Suggestion for COMPATIBILITY.md

### PR DESCRIPTION
**Pull request by carlosrodlop**
Including within the Build steps and post-build actions the Backup plugin

JIRA OSS: https://issues.jenkins-ci.org/browse/JENKINS-34346

@reviewbybees

### CODE REVIEW

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `PerfSigRecorder` and 5 more (`performance-signature-dynatrace`): supported as of 2.0
 - [ ] `StashNotifier` (`stashNotifier`): [issue 92](https://github.com/jenkinsci/stashnotifier-plugin/issues/92)
 - [X] `LambdaUploadBuildStep`, `LambdaInvokeBuildStep`, `LambdaEventSourceBuildStep` (`aws-lambda`): supported as of 0.5.0
+- [ ] `Backup`: [JENKINS-34346](https://issues.jenkins-ci.org/browse/JENKINS-34346)
```
Comment (jglick) [inline COMPATIBILITY.md]: :bug: There is no such build step that I can find.

**Comment by oleg-nenashev** [comment]
I see no problem if somebody implements it.
By the way, Pipeline is not about Jenkins self-administration right now.
For such purpose I'd recommend Scriptler.

File: COMPATIBILITY.md (hunk 1 above)
Comment (carlosrodlop) [inline COMPATIBILITY.md]: My fault, I didn't check the scope of the OSS Backup plugin, it doesn't work at job level so it doesn't make any sense to make it compatible within pipeline. No connection with the AbstractBuild, so it couldn't take part into a Build.

**Comment by carlosrodlop** [comment]
According to my previous comments, I close this pull request.

**Pull request closed without merging**